misc/python/materialize/mzcompose.py

```diff
@@ -581,25 +593,221 @@ def run_compose(
+        )
+        config.update(
+            {
+                "command": command,
+                "ports": [port],
+                "environment": environment,
+                "volumes": volumes,
+            }
+        )
+
+        super().__init__(name=name, config=config)
+
+
+class Zookeeper(PythonService):
+    def __init__(
+        self,
+        name: str = "zookeeper",
+        image: str = f"confluentinc/cp-zookeeper:{DEFAULT_CONFLUENT_PLATFORM_VERSION}",
+        port: int = 2181,
+        environment: List[str] = ["ZOOKEEPER_CLIENT_PORT=2181"],
+    ) -> None:
+        super().__init__(
+            name="zookeeper",
+            config={"image": image, "ports": [port], "environment": environment},
+        )
+
 
-class Mz(PythonService):
-    def __init__(self) -> None:
+class Kafka(PythonService):
+    def __init__(
+        self,
+        name: str = "kafka",
+        image: str = f"confluentinc/cp-kafka:{DEFAULT_CONFLUENT_PLATFORM_VERSION}",
+        port: int = 9092,
+        environment: List[str] = [
+            "KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181",
+            "KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092",
+            "KAFKA_AUTO_CREATE_TOPICS_ENABLE=false",
+            "KAFKA_CONFLUENT_SUPPORT_METRICS_ENABLE=false",
+            "KAFKA_MIN_INSYNC_REPLICAS=1",
+            "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1",
+            "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR=1",
+            "KAFKA_TRANSACTION_STATE_LOG_MIN_ISR=1",
+        ],
+        depends_on: List[str] = ["zookeeper"],
+    ) -> None:
         super().__init__(
-            name="materialized",
+            name=name,
             config={
-                "mzbuild": "materialized",
-                "command": "--data-directory=/share/mzdata --experimental --disable-telemetry",
-                "ports": [6875],
-                "environment": ["MZ_DEV=1"],
+                "image": image,
+                "ports": [port],
+                "environment": environment,
+                "depends_on": depends_on,
+            },
+        )
+
+
+class SchemaRegistry(PythonService):
+    def __init__(
+        self,
+        name: str = "schema-registry",
+        image: str = f"confluentinc/cp-schema-registry:{DEFAULT_CONFLUENT_PLATFORM_VERSION}",
+        port: int = 8081,
+        environment: List[str] = [
+            "SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS=PLAINTEXT://kafka:9092",
+            "SCHEMA_REGISTRY_HOST_NAME=localhost",
+        ],
+        depends_on: List[str] = ["kafka", "zookeeper"],
+    ) -> None:
+        super().__init__(
+            name=name,
+            config={
+                "image": image,
+                "ports": [port],
+                "environment": environment,
+                "depends_on": depends_on,
+            },
+        )
+
+
+class Postgres(PythonService):
+    def __init__(
+        self,
+        name: str = "postgres",
```

Comment:
And I immediately got an automatic email about this line:

```
GitGuardian has detected the following Generic Database Assignment exposed within your GitHub account.
```

Seems like some third-party company is listening on the activity feed and doing security checks/alters. @benesch, do we care about them?